src/environments/environment.ts

```diff
@@ -16,6 +16,8 @@ export const ITEMS_PER_PAGE = 5;
 export const STACK_EXCHANGE_ID = keys.STACK_EXCHANGE_ID;
 export const STACK_EXCHANGE_ACCESS_TOKEN = keys.STACK_EXCHANGE_ACCESS_TOKEN;
 export const AZURE_APP_CONFIGURATION_CONNECTION_STRING = keys.AZURE_APP_CONFIGURATION_CONNECTION_STRING;
+export const DATE_FORMAT = 'yyyy-MM-dd';
+export const DATE_FORMAT_YEAR = 'YYYY-MM-DD';
```

Comment:
Just wondering why we have this two date formats?

Reply:
@josepato87  the project has many string date formats, so create these variables to import the format, and not write it every time in code. yyyy-MM-dd, yyyy-MM-dd, yyyy-MM-dd, etc.

Description formats date: 
DATE_FORMAT = `yyyy`: Year,   `MM`: Month in year,  `dd`: Day in month

DATE_FORMAT_YEAR = `YYYY`:  Week year,   `MM`:  Month in year,  `DD`: Day in year

Date and Time Patterns
![image](https://user-images.githubusercontent.com/12575632/102100394-61164d00-3df7-11eb-9668-729b8d955a47.png)